Makefile

```diff
@@ -1,16 +1,18 @@
 .DEFAULT_GOAL := help
 
 # local-port:service-port
-CONDITION_API_PORT_FW=9001:9001
-CONDITION_ORC_API_PORT_FW=9002:9001
-ALLOY_PORT_FW=9091:9091
-FLEETDB_PORT_FW=8000:8000
-CRDB_UI_PORT_FW=8081:8080
-CRDB_PORT_FW=26257:26257
-CHAOS_DASH_PORT_FW=2333:2333
-JAEGER_DASH_PORT_FW=16686:16686
-OTEL_PORT_FW=4317:4317
-MINIO_PORT_FW=9000:9000
+
+WITH_LAN=
+CONDITION_API_PORT_FW=9001
+CONDITION_ORC_API_PORT_FW=9001
```

Comment:
These two conflict though, so maybe we should keep condition orc `9002:9001`.

Reply:
i think you have proved that my "simple fix" for some things here is not viable. Im going to reject this PR and try again.